### DISCUSSION TITLE
Add the library to nuget

### DISCUSSION THREAD

**Discussion by slize-it** [stephenberry/glaze#1049] [Ideas]
Hey, it would be very handy if you could add this fantastic library to https://www.nuget.org/packages :-)

**Comment by stephenberry** [comment]
I won't be able to maintain a nuget package, but if you're willing to setup the package I'd be happy to merge any pull requests necessary on the Glaze side.

**Comment by stephenberry** [comment]
I'm moving this to a discussion, but feel free to submit a pull request to add support for nuget.
